packages/@guardian/source-react-components-development-kitchen/src/lines/Lines.tsx

```diff
@@ -36,18 +35,42 @@ export interface LinesProps extends Props {
 export const Lines = ({
 	effect = 'straight',
```

Comment:
i think having these configurable as a prop in this way means you'll ship each style even if it's not used. i don't imagine we often need to change them once rendered? would they work as separate components?

Reply:
Yes, they would work great as separate components. I’ve marked this as deprecated in favour of importing the exact `*Line` component needed.

Reply:
nice one 👍